client/src/components/kpi/kpi.tsx

```
@@ -0,0 +1,51 @@
+import Api from "@/api/api";
+import { useQuery } from "@tanstack/react-query";
+import { Spinner } from "../ui/spinner";
+
+import Error from "../Error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)